DEBUG some5 T5 MDC-TEST5 [main] SocketServerTestCase - Message 1
DEBUG some5 T5 MDC-TEST5 [main] root - Message 2
 INFO some5 T5 MDC-TEST5 [main] SocketServerTestCase - Message 3
 WARN some5 T5 MDC-TEST5 [main] SocketServerTestCase - Message 4
LETHAL some5 T5 MDC-TEST5 [main] SocketServerTestCase - Message 5
DEBUG some5 T5 MDC-TEST5 [main] SocketServerTestCase - Message 6
java.lang.Exception: Just testing
	at org.apache.log4j.net.SocketServerTestCase.common(SocketServerTestCase.java:XXX)
	at org.apache.log4j.net.SocketServerTestCase.test5(SocketServerTestCase.java:XXX)
	at java.lang.reflect.Method.invoke(Native Method)
	at junit.framework.TestCase.runTest(TestCase.java:XXX)
	at junit.framework.TestCase.runBare(TestCase.java:XXX)
	at junit.framework.TestResult$1.protect(TestResult.java:XXX)
	at junit.framework.TestResult.runProtected(TestResult.java:XXX)
	at junit.framework.TestResult.run(TestResult.java:XXX)
	at junit.framework.TestCase.run(TestCase.java:XXX)
	at junit.framework.TestSuite.runTest(TestSuite.java:XXX)
	at junit.framework.TestSuite.run(TestSuite.java:XXX)
ERROR some5 T5 MDC-TEST5 [main] root - Message 7
java.lang.Exception: Just testing
	at org.apache.log4j.net.SocketServerTestCase.common(SocketServerTestCase.java:XXX)
	at org.apache.log4j.net.SocketServerTestCase.test5(SocketServerTestCase.java:XXX)
	at java.lang.reflect.Method.invoke(Native Method)
	at junit.framework.TestCase.runTest(TestCase.java:XXX)
	at junit.framework.TestCase.runBare(TestCase.java:XXX)
	at junit.framework.TestResult$1.protect(TestResult.java:XXX)
	at junit.framework.TestResult.runProtected(TestResult.java:XXX)
	at junit.framework.TestResult.run(TestResult.java:XXX)
	at junit.framework.TestCase.run(TestCase.java:XXX)
	at junit.framework.TestSuite.runTest(TestSuite.java:XXX)
	at junit.framework.TestSuite.run(TestSuite.java:XXX)
